pkgs/desktops/gnome-3/core/mutter/default.nix

```diff
@@ -111,6 +111,13 @@ stdenv.mkDerivation rec {
       sha256 = "0imy2j8af9477jliwdq4jc40yw1cifsjjf196gnmwxr9rkj0hbrd";
     })
 
+    # gnome-3-34 2020-04-24
+    # also fixes https://mail.gnome.org/archives/distributor-list/2020-April/msg00001.html
```

Comment:
This has already been fixed in 3.34.2:

https://gitlab.gnome.org/GNOME/mutter/-/commit/f4574ea36f0b2babe21b3ac1ea3c801ca32bc543

Reply:
comment removed